pkg/controller/razeedeployment/razeedeployment_controller.go

```diff
@@ -856,275 +871,308 @@ func (r *ReconcileRazeeDeployment) Reconcile(request reconcile.Request) (reconci
 	}
 
 	/******************************************************************************
-	CREATE THE RAZEE JOB
+	Create watch-keeper deployment,rrs3-controller deployment, apply parent rrs3
 	/******************************************************************************/
-
-	foundJob := &batch.Job{}
-	reqLogger.V(0).Info("Finding job")
+	rrs3Deployment := &appsv1.Deployment{}
+	reqLogger.V(0).Info("Finding Rhm RemoteResourceS3 deployment")
 	err = r.client.Get(context.TODO(), types.NamespacedName{
-		Name:      utils.RAZEE_DEPLOY_JOB_NAME,
+		Name:      utils.RHM_REMOTE_RESOURCE_S3_DEPLOYMENT_NAME,
 		Namespace: request.Namespace,
-	}, foundJob)
+	}, rrs3Deployment)
 	if errors.IsNotFound(err) {
-		reqLogger.V(0).Info("Creating razzeedeploy-job")
-		job := r.makeRazeeJob(request, instance)
-		err = r.client.Create(context.TODO(), job)
+		reqLogger.V(0).Info("Creating RemoteResourceS3 deployment")
+		rrs3Deployment = r.makeRemoteResourceS3Deployment(instance)
+		err = r.client.Create(context.TODO(), rrs3Deployment)
 		if err != nil {
-			reqLogger.Error(err, "Failed to create Job on cluster")
+			reqLogger.Error(err, "Failed to create RemoteResourceS3 deployment on cluster")
 			return reconcile.Result{}, err
 		}
-		reqLogger.Info("job created successfully")
+		reqLogger.Info("RemoteResourceS3 deployment created successfully")
 
-		message := "Razee-Deploy-Job install starting"
+		message := "RemoteResourceS3 deployment install starting"
 		instance.Status.Conditions.SetCondition(status.Condition{
 			Type:    marketplacev1alpha1.ConditionInstalling,
 			Status:  corev1.ConditionTrue,
-			Reason:  marketplacev1alpha1.ReasonRazeeDeployJobStart,
+			Reason:  marketplacev1alpha1.ReasonRhmRemoteResourceS3DeploymentStart,
 			Message: message,
 		})
 
 		_ = r.client.Status().Update(context.TODO(), instance)
 
-		// requeue to grab the "foundJob" and continue to update status
-		// wait 30 seconds so the job has time to complete
-		// not entirely necessary, but the struct on Status.Conditions needs the Conditions in the job to be populated.
-		return reconcile.Result{RequeueAfter: time.Second * 30}, nil
-		// return reconcile.Result{Requeue: true}, nil
-		// return reconcile.Result{}, nil
+		return reconcile.Result{Requeue: true}, nil
+
 	} else if err != nil {
-		reqLogger.Error(err, "Failed to get Job(s) from Cluster")
+		reqLogger.Error(err, "Failed to get RemoteResourceS3 deployment from Cluster")
 		return reconcile.Result{}, err
 	}
 
-	if err := controllerutil.SetControllerReference(instance, foundJob, r.scheme); err != nil {
+	if err := controllerutil.SetControllerReference(instance, rrs3Deployment, r.scheme); err != nil {
 		reqLogger.Error(err, "Failed to set controller reference")
 		return reconcile.Result{}, err
 	}
 
```

Comment:
Please check for changes to the rrs3 deployment and update if necessary. In particular the version may change in the deployment.